app/pages/project/home.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import FinishedBanner from './finished-banner';
+import ProjectMetadata from './metadata';
+import ProjectHomeWorkflowButtons from './home-workflow-buttons';
+import markdownz from 'markdownz';
+const Markdown = markdownz.Markdown;
```

Comment:
Yeah, I had it that way before, but it was throwing some kind of import error or compiling error; I don't recall exactly which and I didn't spend any time on debugging it since getting the refactor done was more important. This seemed to be a working solution for it.